cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/app/etl/realtime/ETLWorker.java

```diff
@@ -183,15 +184,15 @@ private void initializeSource(WorkerContext context) throws Exception {
 
   @SuppressWarnings("unchecked")
   private void initializeSinks(WorkerContext context) throws Exception {
-    List<String> sinkPluginIds = GSON.fromJson(context.getSpecification().getProperty(Constants.Sink.PLUGINIDS),
-                                               STRING_LIST_TYPE);
+    List<SinkInfo> sinkPluginIds = GSON.fromJson(context.getSpecification().getProperty(Constants.Sink.PLUGINIDS),
```

Comment:
pipeline.getRegisterer's signature has changed, so now it returns a List<SinkInfo> instead of List<String>